### PR TITLE
Don't mix quoted and unquoted keys

### PR DESCRIPTION
**Pull request by Alex7Li**
This commit fixes an issue where quoted and unquoted keys were mixed in helper.js. This resulted in the compiler sometimes minimizing the keys and making it impossible to call the parameters from external functions. The exact conditions I can't say, since the compiler here was doing fine and passing tests that I made to reproduce the error I was getting when compiling the src/ folder elsewhere (in measurement library). This fix should ensure consistent behavior everywhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,6 @@ This library provides the ability to process messages passed onto a data layer q
 - [License](#license)
   
 
-<<<<<<< HEAD
```
Comment (calhounsm) [inline README.md:24]: Merge conflict marker?

File: src/helper/data-layer-helper.js [hunk 2]
```diff
@@ -190,7 +193,7 @@ class DataLayerHelper {
       if (arguments.length === 1 && type(arguments[0]) === 'object') {
         merge(arguments[0], model);
       } else if (arguments.length === 2 &&
-        type(arguments[0]) === 'string') {
+          type(arguments[0]) === 'string') {
```
Comment (calhounsm) [inline src/helper/data-layer-helper.js:196]: Nit: wrong indentation.

Comment (Alex7Li) [reply src/helper/data-layer-helper.js]: fixed

File: README.md (hunk 1 above)
Comment (Alex7Li) [reply README.md:24]: Yup, the merge has been resolved

**Pull request merged**